data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -798,6 +798,68 @@
   ]
 },{
   "type" : "recipe",
+  "result": "mutagen_lamia",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_MUTAGEN",
+  "skill_used": "cooking",
+  "skills_required": [ "firstaid", 1 ],
+  "difficulty": 9,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
```

Comment:
Ah shoot, I went through and fixed the entirety of it after leaving my comment, then you commented while I went ahead and made said fixes. >.<